class.wp-cldr.php

```diff
@@ -146,6 +153,7 @@ public function get_cldr_locale( $wp_locale ) {
 		);
 
 		// Convert underscores to dashes and everything to lowercase.
+		$cleaned_up_wp_locale = '';
```

Comment:
yep good.  that was a few lines higher and I moved it down but isn't necessary now. will remove.